### ISSUE TITLE
Focus styles: Audit VA.gov for focus states

### ISSUE THREAD

**Issue by davidakennedy** [department-of-veterans-affairs/vets-design-system-documentation#1313]
## Content

The VA.gov Design System needs updated focus styles for better accessibility because the current styles don't meet color contrast guidelines in the Web Content Accessibility Guidelines. In updating the styles, we want to also think ahead to future versions of WCAG, which likely have more stringent requirements for focus styles.
Specifically, the current focus style fails to meet compliance in the scenario of white button on dark background (e.g. the active search button).

Links:
- [Success Criterion 1.4.11 Non-text Contrast](https://www.w3.org/TR/WCAG21/#non-text-contrast)
- [What's New in WCAG 2.2?](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/)
- [2.4.11 Focus Appearance(AA)](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/#2411-focus-appearance-aa)
- [2.4.12 Focus Not Obscured (Minimum) (AA)](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/#2412-focus-not-obscured-minimum-aa)
- [Testing, Evaluation, and Documentation: Double box-shadow focus styling](https://docs.google.com/document/d/1o-bfmdz6arPju9XCiGn-NR9UpgoN1Vp0EB3cvRGMA3Q/edit?usp=sharing)
- [VA Design System Focus Styles: Questions and Assumptions](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/shared-support/accessibility/projects/design-system-focus-styles/focus-styles-questions-assumptions-requirements.md)
- [Use double box-shadow for focus styling #654](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/654)

## Work to be done

This ticket is to find, collate and review current focus (and related) states in interactive controls on VA.gov.
This will include screenshots of the focus states, and likely some other states that should play nicely with focus: resting, hover, active and disabled. It will include links, buttons and other interactive controls on the site. The review won't include every interactive control, product or state, but a wide sampling to get a sense of the landscape.
This review should include collating the findings into a usable format appropriate for the information that is surfaced. This will likely take the format of a Mural board, paired with some analysis in GitHub documentation. This information should be summarized and synthesized in a way that is able to be presented to those less familiar with the issues and work.

## Acceptance criteria

- [x] Links to artifacts are available
- [x] Findings are collated and summarized in a readable format
- [x] Review meeting has been scheduled

**Comment by davidakennedy** [comment]
I wanted to update this ticket with the work done so far.

- The [focus styles audit](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1663602034370/03497f8dc7ef4caf0b5677b6260188cfe594c5ae?wid=0-1669920646432&sender=ud899eb01a8bd7352dc213970) is complete and available in that link to a Mural board.
- I've [documented my recommendations and audit notes on GitHub](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/shared-support/accessibility/projects/design-system-focus-styles/focus-style-audit-recommendations-notes.md). I plan to add some more details about next steps and my early explorations into color on the [color ticket](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1314).
- I met with @humancompanion-usds and @GnatalieH today to discuss the audit and next steps. They felt like the audit was comprehensive enough to move forward. Matt and Natalie, please add any other details that I missed.

Overall, this work can be marked complete. cc: @caw310
